.github/workflows/build_and_push.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and push Docker Image
+
+on: 
+  push:
+    branches:
+      - master
+
+env:
+  DOCKERHUB_REPOSITORY: weseek/mongodb-awesome-backup
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: crazy-max/ghaction-docker-buildx@v1.0.4
+        with:
+          version: v0.3.1
+
+      - name: Login to Docker Hub
+        run: >
+          echo ${{ secrets.DOCKER_REGISTRY_PASSWORD }} |
+          docker login --username ${{ secrets.DOCKER_REGISTRY_USERNAME }} --password-stdin
+
+      - name: Build and push latest Docker Image
+        run: >
+          docker buildx build
+          --tag ${DOCKERHUB_REPOSITORY}
+          --platform linux/amd64
+          --cache-from type=registry,ref=${DOCKERHUB_REPOSITORY}
+          --cache-to type=inline
+          --push
+          .
```

Comment:
These lines seem to be same as `release` job in release.yml.
Could you commonize it?

Reply:
There is no way to commonize step.

* GitHub action can't trigger another action ([ref](https://help.github.com/en/actions/reference/events-that-trigger-workflows))